crates/astria-conductor/src/executor/tests.rs

```diff
@@ -46,7 +49,9 @@ fn make_state(
     }: MakeState,
 ) -> (StateSender, StateReceiver) {
     let genesis_info = GenesisInfo::try_from_raw(raw::GenesisInfo {
-        rollup_id: Bytes::copy_from_slice(ROLLUP_ID.as_ref()),
+        rollup_id: Some(RawRollupId {
+            inner: Bytes::copy_from_slice(ROLLUP_ID.as_ref()),
+        }),
```

Comment:
```suggestion
        rollup_id: ROLLUP_ID.to_raw(),
```

I believe `ROLLUP_ID` is already a type which can cleanly be converted, replace import above with the non raw.